pilot/pkg/model/service.go

```diff
@@ -719,7 +721,8 @@ type WorkloadInfo struct {
 }
 
 func (i WorkloadInfo) ResourceName() string {
-	return i.Name + "/" + i.Namespace
+	ii, _ := netip.AddrFromSlice(i.Address)
```

Comment:
What happens if I'm subscribed to "1.2.3.4", and for whatever reason I lose control plane connection. While that connection is down, the pod with that IP goes away and a new pod for a different service/namespace/sa comes up with the same IP. 

Do I get all the deltas?